pkg/asset/installconfig/alibabacloud/alibabacloud.go

```diff
@@ -136,3 +141,25 @@ func selectRegion(client *Client) (string, error) {
 	}
 	return selectedRegion, nil
 }
+
+func bypassDeprecation() error {
+	confirmationMsg := "DEPRECATED. Alibaba Cloud is deprecated and will be removed in " +
+		"a future version of the installer. Please reach out to your support " +
```

Comment:
Can we update message to the following:

DEPRECATED. Alibaba Cloud is deprecated and will be removed in a future OpenShift version. Please reach out to your Red Hat Support or Technical Account Manager for more information.

Reply:
Done.